lib/Zonemaster/Backend/DB.pm

```diff
@@ -242,6 +242,8 @@ sub test_progress {
                 $test_id,
             );
         }
+
+        return $progress;
```

Comment:
This is a new return value. What call will use that? Will calling code not be affected?

Reply:
Nope it's not, the function already returns a value (3 lines down). Here we return earlier to avoid an extra call to the database that fetch the current progress value.

Reply:
Thanks for clarification.